include/swift/Subsystems.h

```diff
@@ -274,7 +275,8 @@ namespace swift {
                       std::unique_ptr<SILModule> SILMod,
                       StringRef ModuleName, llvm::LLVMContext &LLVMContext,
                       unsigned StartElem = 0,
-                      llvm::GlobalVariable **outModuleHash = nullptr);
+                      llvm::GlobalVariable **outModuleHash = nullptr,
+                      UnifiedStatsReporter *Stats = nullptr);
```

Comment:
Ah, nice catch! I'd forgotten I was also stashing it there. Updated with simpler patch.